src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write code here
+const salaryList = [...document.querySelectorAll('li')];
+
+// функція для перетворення зарплати з рядка в число
+function parseSalary(salaryStr) {
+  return parseInt(salaryStr.replace(/[$,]/g, ''), 10);
+}
+
+function getEmployees(list) {
+  return list.map((item) => ({
+    name: item.textContent.trim(),
+    position: item.getAttribute('data-position'),
+    salary: parseSalary(item.getAttribute('data-salary')),
+    age: parseInt(item.getAttribute('data-age'), 10),
+  }));
+}
+
+function sortList(employees) {
+  return employees.sort((a, b) => b.salary - a.salary);
+}
+
+const employe = getEmployees(salaryList);
+
+const sortedEmployees = sortList(employe);
```

Comment:
The variable 'employe' is used here, but it should be 'employees' to match the corrected variable name from line 23.